connector.go

```diff
@@ -173,6 +173,13 @@ type AWSReader interface {
 //
 // The connections are not all established while instancing, but the various sessions are, this way connections are only
 // made for services that are called, otherwise only the sessions remain.
+//
+// An error is returned if any of the needed AWS request for creating the reader returns an AWS error, in such case it
+// will have any of the common error codes (see below) or EmptyStaticCreds code or a go standard error in case that no
+// regions are matched with the ones available, at the time, in AWS.
+// See:
+//	* https://docs.aws.amazon.com/AWSEC2/latest/APIReference/errors-overview.html#CommonErrors
+//  * https://docs.aws.amazon.com/STS/latest/APIReference/CommonErrors.html
```

Comment:
Remove the space or add it on the first one hehe

Reply:
I couldn't see it visual in my editor, basically, the first one was a tab and the second 2 spaces; I wanted to have 2 spaces, so I updated the first one.